compiler/vertex-desc.json

```diff
@@ -1779,6 +1779,22 @@
       }
     }
   },
+
```

Comment:
We don't use empty lines between blocks in vertex-desc.json

Reply:
Fixed.